lib/debuglet.js

```diff
@@ -229,9 +227,11 @@ Debuglet.prototype.scheduleBreakpointFetch_ = function(seconds) {
  */
 Debuglet.prototype.updateActiveBreakpoints_ = function(breakpoints) {
   var that = this;
-
   var updatedBreakpointMap = this.convertBreakpointListToMap_(breakpoints);
-  that.logger_.breakpoints(Logger.INFO, 'Server breakpoints:', updatedBreakpointMap);
+
+  if (breakpoints.length) {
+    that.logger_.breakpoints(Logger.INFO, 'Server breakpoints:', updatedBreakpointMap);
```

Comment:
I'm preserving behavior with this PR. We weren't printing this before either. Also, the lack of output is pretty good signal at the INFO level. Printing the full response from the server would be more appropriate at DEBUG level.